### PR TITLE
Update Jenkins plugin parent

### PR DESCRIPTION
**Pull request by scawley-r7**

### CODE REVIEW

File: src/main/java/com/rapid7/jenkins/LogDecorator.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ private void processLine(byte[] bytes, int length) throws IOException {
             }
             // TODO Verify that the byte are encoded using the platform default (not UTF8)
             if (end > 0) {
-                logWriter.writeLogentry(new String(bytes, 0, end + 1));
+                logWriter.writeLogentry(new String(bytes, 0, end + 1, Charset.forName("ascii")));
```
Comment (stopal-r7) [inline src/main/java/com/rapid7/jenkins/LogDecorator.java]: why are we limiting this to ascii only?

Comment (scawley-r7) [reply src/main/java/com/rapid7/jenkins/LogDecorator.java]: build fails if we don't supply a charset, not 100% sure what this should be though @miguelcasanova  you might have a better idea on this?

Comment (stopal-r7) [reply src/main/java/com/rapid7/jenkins/LogDecorator.java]: i think we  can use `UTF-8` the as in `ascii` case it will support only english logs

Comment (danilosterrapid7) [reply src/main/java/com/rapid7/jenkins/LogDecorator.java]: Can we assume that we are only going to support english logs? Other languages would have special chars such as `Perdão, Até, många, några, är`.

File: src/main/resources/com/rapid7/jenkins/Messages.properties [hunk 2]
```diff
@@ -1 +1 @@
-DisplayName = Rapid7 Insight Forwarder
+displayName = Rapid7 Insight Forwarder
```
Comment (danilosterrapid7) [inline src/main/resources/com/rapid7/jenkins/Messages.properties]: Should it be renamed to `displayName = Rapid7 Insight Log Forwarder`?

**Pull request merged**